src/editors/containers/TextEditor/index.jsx

```diff
@@ -86,11 +89,16 @@ export const TextEditor = ({
     >
       <div className="editor-body h-75 overflow-auto">
         <ImageUploadModal
-          isOpen={isOpen}
```

Comment:
Note: there is a lot of changing image modal open/closed items to reference that they are related to images in their name.